scripts/package

```diff
@@ -1,6 +1,17 @@
 #!/bin/bash
 set -e
 
+extract_latest_version_tgz() {
+    # Takes in a path to a chart's directory, finds the latest version's tgz and extracts it
+    # All crd tgz are ignored
+    # Max depth is set to prevent extracting a tgz contained within another tgz, which is the case for charts containing a helm repo
```

Comment:
This should cover sub-charts as long as they are not packaged into a tgz like `parent-chart/charts/subchart.tgz`, which I don't think we do since all the sub-charts I found were in their own separate directories as `parent-chart/charts/subchart/` after extraction. 
The main reason I implemented this constraint is because OPA's gatekeeper has a helm repo inside the chart. It has an index.yaml and a tgz which will get extracted and non-mirrored images will get picked. Right now, our process would only allow deleting that tgz through the patch which will pretty much make it unreadable.